app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/AttachmentOptions.kt

```diff
@@ -248,6 +229,14 @@ private fun buildAttachmentOptionItems(
                     R.drawable.ic_video
                 ) { captureVideoFlow?.launch() }
             )
+            add(
+                AttachmentOptionItem(
+                    isFileSharingEnabled,
+                    R.string.attachment_record_video,
+                    R.drawable.ic_video,
+                    onRecordAudioMessageClicked
+                )
```

Comment:
It's better to merge something clean and correct to develop. We are not sure how long it would take until sending audio messages feature is merged to develop to fix the typo and icon